Delete todo-app

ecraig12345 opened the pull request:
The todo demo app for `@fluentui/react` is very outdated and not so useful for reference (if anybody could even find it, which is also unlikely). I previously made a PR #15828 fixing some of the issues, but I think especially with convergence work in progress using very different patterns, it's best to just get rid of the package and maybe add back something better in the future.

fabricteam commented:
## [Perf Analysis (`@fluentui/react`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/Avatar-mount_base.html">838</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/Avatar-mount.html">849</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>BaseButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/BaseButton-mount_base.html">981</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/BaseButton-mount.html">993</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Breadcrumb</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/Breadcrumb-mount_base.html">2708</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/Breadcrumb-mount.html">2715</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ButtonNext</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/ButtonNext-mount_base.html">573</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/ButtonNext-mount.html">566</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Checkbox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/Checkbox-mount_base.html">1694</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/Checkbox-mount.html">1707</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>CheckboxBase</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/CheckboxBase-mount_base.html">1483</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/CheckboxBase-mount.html">1441</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ChoiceGroup</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/ChoiceGroup-mount_base.html">5275</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/ChoiceGroup-mount.html">5337</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ComboBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/ComboBox-mount_base.html">1062</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/ComboBox-mount.html">1072</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>CommandBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/CommandBar-mount_base.html">10764</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/CommandBar-mount.html">10596</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/ContextualMenu-mount_base.html">6427</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/ContextualMenu-mount.html">6495</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DefaultButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/DefaultButton-mount_base.html">1228</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/DefaultButton-mount.html">1195</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRow</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/DetailsRow-mount_base.html">4028</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/DetailsRow-mount.html">3990</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowFast</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/DetailsRowFast-mount_base.html">3925</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/DetailsRowFast-mount.html">3980</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowNoStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/DetailsRowNoStyles-mount_base.html">3815</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/DetailsRowNoStyles-mount.html">3768</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Dialog</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/Dialog-mount_base.html">2243</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/Dialog-mount.html">2287</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DocumentCardTitle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/DocumentCardTitle-mount_base.html">160</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/DocumentCardTitle-mount.html">156</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Dropdown</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/Dropdown-mount_base.html">3510</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/Dropdown-mount.html">3496</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusTrapZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/FocusTrapZone-mount_base.html">1846</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/FocusTrapZone-mount.html">1900</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/FocusZone-mount_base.html">1851</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/FocusZone-mount.html">1852</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>IconButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/IconButton-mount_base.html">1947</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/IconButton-mount.html">1907</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Label</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/Label-mount_base.html">351</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/Label-mount.html">358</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Layer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/Layer-mount_base.html">1939</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/Layer-mount.html">2074</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Link</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/Link-mount_base.html">514</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/Link-mount.html">512</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/MakeStyles-mount_base.html">1862</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/MakeStyles-mount.html">1819</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/MenuButton-mount_base.html">1621</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/MenuButton-mount.html">1596</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MessageBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/MessageBar-mount_base.html">2136</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/MessageBar-mount.html">2183</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Nav</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/Nav-mount_base.html">3436</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/Nav-mount.html">3530</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>OverflowSet</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/OverflowSet-mount_base.html">1101</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/OverflowSet-mount.html">1147</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Panel</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/Panel-mount_base.html">2212</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/Panel-mount.html">2258</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/Persona-mount_base.html">885</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/Persona-mount.html">916</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Pivot</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/Pivot-mount_base.html">1490</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/Pivot-mount.html">1492</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>PrimaryButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/PrimaryButton-mount_base.html">1398</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/PrimaryButton-mount.html">1401</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Rating</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/Rating-mount_base.html">8348</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/Rating-mount.html">8435</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SearchBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/SearchBox-mount_base.html">1490</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/SearchBox-mount.html">1408</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Shimmer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/Shimmer-mount_base.html">2741</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/Shimmer-mount.html">2717</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Slider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/Slider-mount_base.html">2095</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/Slider-mount.html">2150</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/SpinButton-mount_base.html">5294</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/SpinButton-mount.html">5512</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Spinner</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/Spinner-mount_base.html">431</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/Spinner-mount.html">427</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SplitButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/SplitButton-mount_base.html">3479</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/SplitButton-mount.html">3465</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Stack</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/Stack-mount_base.html">572</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/Stack-mount.html">610</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithIntrinsicChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/StackWithIntrinsicChildren-mount_base.html">1757</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/StackWithIntrinsicChildren-mount.html">1790</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithTextChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/StackWithTextChildren-mount_base.html">5030</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/StackWithTextChildren-mount.html">5036</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchColorPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/SwatchColorPicker-mount_base.html">10824</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/SwatchColorPicker-mount.html">10952</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Tabs</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/Tabs-mount_base.html">1514</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/Tabs-mount.html">1522</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>TagPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/TagPicker-mount_base.html">2684</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/TagPicker-mount.html">2629</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TeachingBubble</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/TeachingBubble-mount_base.html">12514</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/TeachingBubble-mount.html">12349</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Text</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/Text-mount_base.html">448</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/Text-mount.html">450</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/TextField-mount_base.html">1503</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/TextField-mount.html">1531</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/ThemeProvider-mount_base.html">1255</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/ThemeProvider-mount.html">1252</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/ThemeProvider-virtual-rerender_base.html">614</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/ThemeProvider-virtual-rerender.html">607</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProviderNext</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/ThemeProviderNext-mount_base.html">6902</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/ThemeProviderNext-mount.html">6974</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Toggle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/Toggle-mount_base.html">847</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/Toggle-mount.html">855</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>buttonNative</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/buttonNative-mount_base.html">118</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test/buttonNative-mount.html">116</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

## Perf Analysis (`@fluentui/react-northstar`)
<details><summary>Perf tests with no regressions</summary>

Scenario | Current PR Ticks | Baseline Ticks | Ratio
:--- | ---:| ---:| ---:
TreeWith60ListItems.default | [203](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/TreeWith60ListItems.default.html) | [180](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/TreeWith60ListItems.default_base.html) | 1.13:1
BoxMinimalPerf.default | [409](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/BoxMinimalPerf.default.html) | [368](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/BoxMinimalPerf.default_base.html) | 1.11:1
AttachmentMinimalPerf.default | [180](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/AttachmentMinimalPerf.default.html) | [163](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/AttachmentMinimalPerf.default_base.html) | 1.1:1
FlexMinimalPerf.default | [331](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/FlexMinimalPerf.default.html) | [313](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/FlexMinimalPerf.default_base.html) | 1.06:1
CardMinimalPerf.default | [654](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/CardMinimalPerf.default.html) | [622](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/CardMinimalPerf.default_base.html) | 1.05:1
ChatDuplicateMessagesPerf.default | [341](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/ChatDuplicateMessagesPerf.default.html) | [325](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/ChatDuplicateMessagesPerf.default_base.html) | 1.05:1
DividerMinimalPerf.default | [438](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/DividerMinimalPerf.default.html) | [416](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/DividerMinimalPerf.default_base.html) | 1.05:1
GridMinimalPerf.default | [384](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/GridMinimalPerf.default.html) | [364](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/GridMinimalPerf.default_base.html) | 1.05:1
PopupMinimalPerf.default | [619](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/PopupMinimalPerf.default.html) | [589](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/PopupMinimalPerf.default_base.html) | 1.05:1
ProviderMinimalPerf.default | [1107](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/ProviderMinimalPerf.default.html) | [1052](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/ProviderMinimalPerf.default_base.html) | 1.05:1
AttachmentSlotsPerf.default | [1291](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/AttachmentSlotsPerf.default.html) | [1237](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/AttachmentSlotsPerf.default_base.html) | 1.04:1
CarouselMinimalPerf.default | [535](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/CarouselMinimalPerf.default.html) | [515](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/CarouselMinimalPerf.default_base.html) | 1.04:1
DropdownManyItemsPerf.default | [831](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/DropdownManyItemsPerf.default.html) | [802](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/DropdownManyItemsPerf.default_base.html) | 1.04:1
ListWith60ListItems.default | [726](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/ListWith60ListItems.default.html) | [697](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/ListWith60ListItems.default_base.html) | 1.04:1
TableMinimalPerf.default | [472](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/TableMinimalPerf.default.html) | [456](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/TableMinimalPerf.default_base.html) | 1.04:1
TooltipMinimalPerf.default | [1092](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/TooltipMinimalPerf.default.html) | [1054](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/TooltipMinimalPerf.default_base.html) | 1.04:1
AnimationMinimalPerf.default | [457](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/AnimationMinimalPerf.default.html) | [444](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/AnimationMinimalPerf.default_base.html) | 1.03:1
CheckboxMinimalPerf.default | [3057](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/CheckboxMinimalPerf.default.html) | [2968](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/CheckboxMinimalPerf.default_base.html) | 1.03:1
LabelMinimalPerf.default | [437](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/LabelMinimalPerf.default.html) | [426](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/LabelMinimalPerf.default_base.html) | 1.03:1
ListNestedPerf.default | [629](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/ListNestedPerf.default.html) | [611](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/ListNestedPerf.default_base.html) | 1.03:1
ReactionMinimalPerf.default | [436](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/ReactionMinimalPerf.default.html) | [423](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/ReactionMinimalPerf.default_base.html) | 1.03:1
SegmentMinimalPerf.default | [389](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/SegmentMinimalPerf.default.html) | [377](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/SegmentMinimalPerf.default_base.html) | 1.03:1
SkeletonMinimalPerf.default | [402](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/SkeletonMinimalPerf.default.html) | [391](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/SkeletonMinimalPerf.default_base.html) | 1.03:1
CustomToolbarPrototype.default | [4241](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/CustomToolbarPrototype.default.html) | [4117](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/CustomToolbarPrototype.default_base.html) | 1.03:1
ToolbarMinimalPerf.default | [1083](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/ToolbarMinimalPerf.default.html) | [1055](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/ToolbarMinimalPerf.default_base.html) | 1.03:1
VideoMinimalPerf.default | [725](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/VideoMinimalPerf.default.html) | [705](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/VideoMinimalPerf.default_base.html) | 1.03:1
AccordionMinimalPerf.default | [172](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/AccordionMinimalPerf.default.html) | [169](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/AccordionMinimalPerf.default_base.html) | 1.02:1
AlertMinimalPerf.default | [312](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/AlertMinimalPerf.default.html) | [305](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/AlertMinimalPerf.default_base.html) | 1.02:1
ButtonMinimalPerf.default | [198](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/ButtonMinimalPerf.default.html) | [195](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/ButtonMinimalPerf.default_base.html) | 1.02:1
DropdownMinimalPerf.default | [3399](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/DropdownMinimalPerf.default.html) | [3317](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/DropdownMinimalPerf.default_base.html) | 1.02:1
LayoutMinimalPerf.default | [420](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/LayoutMinimalPerf.default.html) | [412](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/LayoutMinimalPerf.default_base.html) | 1.02:1
LoaderMinimalPerf.default | [745](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/LoaderMinimalPerf.default.html) | [727](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/LoaderMinimalPerf.default_base.html) | 1.02:1
ProviderMergeThemesPerf.default | [1739](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/ProviderMergeThemesPerf.default.html) | [1712](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/ProviderMergeThemesPerf.default_base.html) | 1.02:1
RadioGroupMinimalPerf.default | [523](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/RadioGroupMinimalPerf.default.html) | [511](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/RadioGroupMinimalPerf.default_base.html) | 1.02:1
RefMinimalPerf.default | [247](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/RefMinimalPerf.default.html) | [242](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/RefMinimalPerf.default_base.html) | 1.02:1
StatusMinimalPerf.default | [758](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/StatusMinimalPerf.default.html) | [746](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/StatusMinimalPerf.default_base.html) | 1.02:1
TreeMinimalPerf.default | [884](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/TreeMinimalPerf.default.html) | [868](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/TreeMinimalPerf.default_base.html) | 1.02:1
AvatarMinimalPerf.default | [223](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/AvatarMinimalPerf.default.html) | [221](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/AvatarMinimalPerf.default_base.html) | 1.01:1
DialogMinimalPerf.default | [866](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/DialogMinimalPerf.default.html) | [854](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/DialogMinimalPerf.default_base.html) | 1.01:1
HeaderSlotsPerf.default | [877](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/HeaderSlotsPerf.default.html) | [866](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/HeaderSlotsPerf.default_base.html) | 1.01:1
ImageMinimalPerf.default | [436](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/ImageMinimalPerf.default.html) | [431](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/ImageMinimalPerf.default_base.html) | 1.01:1
ItemLayoutMinimalPerf.default | [1402](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/ItemLayoutMinimalPerf.default.html) | [1388](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/ItemLayoutMinimalPerf.default_base.html) | 1.01:1
MenuMinimalPerf.default | [948](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/MenuMinimalPerf.default.html) | [941](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/MenuMinimalPerf.default_base.html) | 1.01:1
SliderMinimalPerf.default | [1662](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/SliderMinimalPerf.default.html) | [1650](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/SliderMinimalPerf.default_base.html) | 1.01:1
SplitButtonMinimalPerf.default | [4073](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/SplitButtonMinimalPerf.default.html) | [4033](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/SplitButtonMinimalPerf.default_base.html) | 1.01:1
TableManyItemsPerf.default | [2213](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/TableManyItemsPerf.default.html) | [2189](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/TableManyItemsPerf.default_base.html) | 1.01:1
TextAreaMinimalPerf.default | [594](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/TextAreaMinimalPerf.default.html) | [590](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/TextAreaMinimalPerf.default_base.html) | 1.01:1
ButtonOverridesMissPerf.default | [1836](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/ButtonOverridesMissPerf.default.html) | [1844](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/ButtonOverridesMissPerf.default_base.html) | 1:1
ButtonSlotsPerf.default | [603](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/ButtonSlotsPerf.default.html) | [603](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/ButtonSlotsPerf.default_base.html) | 1:1
ChatWithPopoverPerf.default | [412](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/ChatWithPopoverPerf.default.html) | [413](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/ChatWithPopoverPerf.default_base.html) | 1:1
HeaderMinimalPerf.default | [422](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/HeaderMinimalPerf.default.html) | [422](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/HeaderMinimalPerf.default_base.html) | 1:1
ListCommonPerf.default | [712](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/ListCommonPerf.default.html) | [709](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/ListCommonPerf.default_base.html) | 1:1
ListMinimalPerf.default | [565](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/ListMinimalPerf.default.html) | [566](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/ListMinimalPerf.default_base.html) | 1:1
IconMinimalPerf.default | [670](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/IconMinimalPerf.default.html) | [668](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/IconMinimalPerf.default_base.html) | 1:1
ChatMinimalPerf.default | [691](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/ChatMinimalPerf.default.html) | [698](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/ChatMinimalPerf.default_base.html) | 0.99:1
DatepickerMinimalPerf.default | [6038](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/DatepickerMinimalPerf.default.html) | [6074](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/DatepickerMinimalPerf.default_base.html) | 0.99:1
EmbedMinimalPerf.default | [4437](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/EmbedMinimalPerf.default.html) | [4493](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/EmbedMinimalPerf.default_base.html) | 0.99:1
InputMinimalPerf.default | [1339](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/InputMinimalPerf.default.html) | [1351](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/InputMinimalPerf.default_base.html) | 0.99:1
MenuButtonMinimalPerf.default | [1732](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/MenuButtonMinimalPerf.default.html) | [1749](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/MenuButtonMinimalPerf.default_base.html) | 0.99:1
FormMinimalPerf.default | [462](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/FormMinimalPerf.default.html) | [480](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/FormMinimalPerf.default_base.html) | 0.96:1
RosterPerf.default | [1279](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/RosterPerf.default.html) | [1337](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/RosterPerf.default_base.html) | 0.96:1
TextMinimalPerf.default | [389](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/TextMinimalPerf.default.html) | [404](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/TextMinimalPerf.default_base.html) | 0.96:1
PortalMinimalPerf.default | [173](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/PortalMinimalPerf.default.html) | [187](https://fluentuipr.z22.web.core.windows.net/pull/18544/perf-test-northstar/PortalMinimalPerf.default_base.html) | 0.93:1

</details><div id="perfComment9423"/>

ling1726 reviewed (approved):
I think that you need to delete it from `nx.json` too, is that right @Hotell  ?

Hotell commented:
hmm while I'm always happy to see code being removed, cant we keep it for v8 "demo" purposes (right you mentioned right to find etc...) ? 

What's our approach with v8 - meaning, some user would like to see a demo app with v8, where should we point them?

ecraig12345 commented:
> hmm while I'm always happy to see code being removed, cant we keep it for v8 "demo" purposes (right you mentioned right to find etc...) ?
> 
> What's our approach with v8 - meaning, some user would like to see a demo app with v8, where should we point them?

Right now, there are no links to the demo code either on our website or in our wiki. And it's debatable whether this app even provides a decent demo of best practices at this point:
- Uses class components and other old patterns. If we're bothering to have a demo app, it should show proper use of function components and hooks.
- Uses scss styling. Though it's a bit ambiguous whether this is actually a pro or con now--last year I would have said this was a clear "con," but now that we're about to move away from v8's CSS-in-JS patterns I'm not sure what we should be recommending to new v8 users.
- Data layer is needlessly complicated (detracts from component demo) and likely not up to date with current best practices in the ecosystem (it's been so long since I've written an app that I don't even know what pattern it "should" be using)

My other PR addressed these issues to some degree, but I just don't have time to get it to a point where I'm confident that it's truly a demo worth pointing people to. (It addressed the class/function component issue, but I'm not sure whether the styling approach I took is actually better or worse, and still not sure about the data layer changes.)

Hotell commented:
> Uses class components and other old patterns. If we're bothering to have a demo app, it should show proper use of function components and hooks.

Well Fabric is build with class components no ? 

> CSS-in-JS patterns I'm not sure what we should be recommending to new v8 users.

gotcha thx

> Data layer is needlessly complicated 

agree


Ok so let's remove it. We should still answer this question though: 

`What's our approach with v8 - meaning, some user would like to see a demo app with v8, where should we point them?`

If the answer is "we don't have anything to show yet", that's fine by me, just so we know how to approach customers.

thx!
